ui/imports/Themes/LightTheme.qml

```diff
@@ -55,6 +55,7 @@ Theme {
     property color buttonSecondaryColor: darkGrey
     property color buttonDisabledForegroundColor: buttonSecondaryColor
     property color buttonDisabledBackgroundColor: grey
+    property color buttonWarnBackgroundColor: "#FFEAEE"
```

Comment:
You'll need to add this color to the dark theme as well. For now, you can use the same hex color

Reply:
ok, I see